test/run-test.rb

```diff
@@ -3,6 +3,7 @@
 $VERBOSE = true
 
 require "pathname"
+require "bundler/setup"
```

Comment:
I'm sorry. I was running the test with `ruby test/run-test.rb`. Added due to loading error of `datasets`.

However, the test is provided in the rake task. I think that it would be better to write to the README to run it there.